definitions/control-v1.yaml

```diff
@@ -3801,6 +3807,251 @@ components:
       - requestMode
       - source
       - target    
+    pulsar_token_auth:
+      type: object
+      description: Pulsar supports authenticating clients using security tokens that are based on JSON Web Tokens.
```

Comment:
This doesn't appear as the description for `authentication`. You can manually add it at as a description in situ e.g. line 3876

I know this is coming from the spec as well but it looks odd that Kafka uses `auth` and Pulsar uses `authentication`.

Reply:
> This doesn't appear as the description for authentication. You can manually add it at as a description in situ e.g. line 3876

Well spotted. Fixed.

> I know this is coming from the spec as well but it looks odd that Kafka uses auth and Pulsar uses authentication.

Well spotted again. I'll raise a bug report for this. https://ably.atlassian.net/browse/WEB-2103

Reply:
Just need to do this one for PATCH as well I think :-)

Reply:
ahh, quite right!